tgui/packages/tgui/styles/main.scss

```diff
@@ -76,6 +76,7 @@
 @include meta.load-css('./interfaces/Fabricator.scss');
 @include meta.load-css('./interfaces/Emojipedia.scss');
 @include meta.load-css('./interfaces/NtosNotepad.scss');
+@include meta.load-css('./interfaces/NtosMessenger.scss');
```

Comment:
ABC order (someone fucked it up so plz just fix the rest of them, there's a nice VSC extension called Sort lines to do this)